snuba/state/__init__.py

```diff
@@ -92,9 +98,13 @@ def wrapper() -> Any:
         return wrapper
 
 
-def numeric(value: Any) -> Any:
+def get_typed_value(value: Any) -> Any:
+    # Return the given value based on its correct type
+    # It supports the following types: int, float, string
+    if value is None:
+        return None
     try:
-        assert isinstance(value, (str, SupportsInt))
+        assert isinstance(value, (str, int))
```

Comment:
Any reason for this change ?

Reply:
I assume to avoid a float being cast to an int here, as float implements __int__